src/docs/environment.mdx

```diff
@@ -164,3 +164,122 @@ If you would like to be able to run `devserver` outside of your root checkout, y
   <em>and</em> port are required in order for DNS and some pages urls to be
   displayed correctly.
 </Alert>
+
+
+## Troubleshooting
+
+**Problem:** something like `pkg_resources.DistributionNotFound: The 'some_dependency<0.6.0,>=0.5.5' distribution was not found and is required by sentry`
+
+**Solution:** Your virtualenv needs to be updated. In sentry, type `make install-sentry-dev`. In getsentry, type `make install-develop-requirements`.
+
+---
+
+**Problem:** You see `Error occured while trying to proxy to: dev.getsentry.net:8000/`
+
+**Solution:**  You likely need to upgrade your Python dependencies. Go to the git root directory and run `make install-sentry-dev`.
+
+---
+
+**Problem:**  "out of memory" issues after running the sentry or getsentry devserver, or during webpacking.
+
+**Solution:**
+
+```bash
+export NODE_OPTIONS="--max-old-space-size=4096"
+```
+
+---
+
+**Problem:** `Module not found: Error: Can't resolve 'integration-docs-platforms'`
+
+**Solution:**
+
+```bash
+make build-platform-assets
+```
+
+---
+
+**Problem:** `SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 76`
+
+Also, another symptom indicating you need to upgrade chromedriver:
+
+```shell
+Traceback (most recent call last):
+  File "/Users/joshua.li/dev/sentry/sentry/src/sentry/utils/pytest/selenium.py", line 344, in browser
+    driver = start_chrome(**chrome_args)
+  File "/Users/joshua.li/dev/sentry/sentry/src/sentry/utils/retries.py", line 41, in execute_with_retry
+    return retrier(functools.partial(fn, *args, **kwargs))
+  File "/Users/joshua.li/dev/sentry/sentry/src/sentry/utils/retries.py", line 85, in __call__
+    error,
+RetryException: Could not successfully execute <functools.partial object at 0x10f31e7e0> within 15.830 seconds (12 attempts.)
+```
+
+**Solution:**
+
+```bash
+brew cask upgrade chromedriver
+```
+
+---
+
+**Problem:**
+
+```shell
+Traceback (most recent call last):
+  ...
+django.core.exceptions.ImproperlyConfigured:
+Application labels aren't unique, duplicates: okta
+make: *** [develop] Error 1
+```
+
+**Solution:**
+
+```jsx
+pip uninstall sentry-auth-github sentry-auth-saml2
```

Comment:
This was a workaround mentioned in an email long ago during the great sentry-plugins migration, I'm fairly sure at this point that no one would have this problem. @scefali you think this can be removed?

Reply:
@joshuarli yea, we no longer use those libs

Reply:
@markstory ^ just in case you missed this

Reply:
@joshuarli My bad, I read this over again. Even though we don't use the packages, the error in question only happens when you have duplicate installs. It's definitely possible that someone could still have this problem I think, though maybe less likely.

Reply:
Probably way less likely, if I had to guess. We could remove this in the near future then.

Reply:
Cool I'll remove this.